Support/Scripts/prepare-android-ndk.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+##
+## Copyright (c) 2014-present, Facebook, Inc.
+## All rights reserved.
+##
+## This source code is licensed under the University of Illinois/NCSA Open
+## Source License found in the LICENSE file in the root directory of this
+## source tree. An additional grant of patent rights can be found in the
+## PATENTS file in the same directory.
+##
+
+import sys
```

Comment:
Sort.

Reply:
What's the rule? Flake8, black, mypy and pyre all passed on sorting them.